pages/common/git-whatchanged.md

```diff
@@ -0,0 +1,17 @@
+# git what-changed
+
+> Git command used to show what has changed with recent commits or files.
+> git what-changed is similar to git log.
+> More information <https://git-scm.com/docs/git-whatchanged>.
+
+- Display full output of recent commits:
+
+`git what-changed`
+
+- Display out of recent commits within a time frame (E.G. 2 hours ago):
+
+`git what-changed --since="2 hours ago"`
+
+- Display changes to selected file(s):
+
+`git what-changed {{file}}`
```

Comment:
I agree with the specified part however I don't feel you need to have path

It needs to be something like 

{{./path/to/file/or/folder}}
the dot is needed because you are already in a git repo and git wouldn't work without it since it would give 
fatal: not a git repository (or any of the parent directories): .git


seems easier to let the user figure out they need to cd into the repo 

can you shorten this "{{./path/to/file/or/folder}}"

Reply:
I'm not sure what you mean @CleanMachine1.  The `./` isn't needed — I just tried it and it worked fine with `path/to/file`.

Reply:
We always do `path/to/file`, unless (1) there's a lot of args, or (2) a simple relative path wouldn't work (3) only a filename (without directories) can be used.

Reply:
not having the / before should work,
so 
either 

./path/to/directory/or/file
or 
path/to/directory/or/file
thats my thoughts on it

it shouldn't have a / to begin since that will start in the root directory /

Reply:
added in commit